src/components/EditorSideBar/EditorSideBarContent.tsx

```diff
@@ -30,7 +38,7 @@ const SideBarContainer = (props: Props) => {
         w="56px"
         bg="background.secondary"
         clipPath="polygon(0% 0%, 100% 8%, 100% 92%, 0% 100%)"
-        z-index="1"
+        zIndex="-1"
```

Comment:
What effect did you want to achieve by adding z-index here

Reply:
It's a sidebar close button, I need to put it -1, otherwise, it will be on top of the normal sidebar content box.